features/org.wso2.carbon.sts.connector.feature/resources/p2.inf

```diff
@@ -0,0 +1,5 @@
+instructions.configure = \
+org.eclipse.equinox.p2.touchpoint.natives.mkdir(path:${installFolder}/../../deployment/);\
+org.eclipse.equinox.p2.touchpoint.natives.mkdir(path:${installFolder}/../../deployment/client/);\
+org.eclipse.equinox.p2.touchpoint.natives.mkdir(path:${installFolder}/../../deployment/client/modules/);\
+org.eclipse.equinox.p2.touchpoint.natives.copy(source:${installFolder}/../features/org.wso2.carbon.sts.connector_${feature.version}/rampart-1.6.1-wso2v43.mar,target:${installFolder}/../../deployment/client/modules/rampart-1.6.1-wso2v43.mar,overwrite:true);\
```

Comment:
Why can't we define the rampart version here as `rampart.wso2.version`?

Reply:
Tried to add the version as rampart.wso2.version but it doesn't pick it up.